routers/web/web.go

```diff
@@ -334,7 +355,7 @@ func registerRoutes(m *web.Route) {
 		m.Get("/users", explore.Users)
 		m.Get("/users/sitemap-{idx}.xml", sitemapEnabled, explore.Users)
 		m.Get("/organizations", explore.Organizations)
-		m.Get("/code", explore.Code)
+		m.Get("/code", reqUnitAccess(unit.TypeCode, perm.AccessModeRead), explore.Code)
```

Comment:
@lunny I can no longer access `/explore/code` page on my local instance because of this change, despite being an admin. What is the reason for this change and where can one grant access to this global code search?

Reply:
Ah, it should be a bug. #25173 should fix it.